org.blender.Blender.json

```diff
@@ -42,7 +42,7 @@
                 {
                     "type": "git",
                     "url": "https://code.videolan.org/videolan/x264.git",
-                    "commit": "d198931a63049db1f2c92d96c34904c69fde8117"
+                    "branch": "stable"
```

Comment:
You really can't randomly bump x264 like that and hope ffmpeg will always like it.

Reply:
Then that's removed